ReactWindows/ReactNative.Shared/Bridge/IReactContext.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// Context wrapper for the React instance to manage lifecycle events.
+    /// </summary>
+    public interface IReactContext : IDisposable

```

Comment:
This is a case where I needed to encapsulate a transitive dependency on UWP. The concrete ReactContext uses the static DispatcherHelpers, which is using Windows.* namespace types. I needed to encapsulate those UWP dependencies that were implementation specific so I could move the other types over needed by the small tests I made portable.

Later, the pure interface will help with dynamic mocking of ReactContext instances when testing objects that take them as a parameter.

Another option would have been to make DispatchHelpers an instance class (versus static, like now), extract an interface, and parameterize that. Since DispatchHelpers is a truly stateless utility object, it seems an appropriate use of a static class, and that didn't seem as good of a choice as a result.